example/src/release_tests/twoFingerPan/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet, View } from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+} from 'react-native-reanimated';
+
+import React from 'react';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+
+const BOX_SIZE = 270;
+
+const clampColor = (v: number) => Math.min(255, Math.max(0, v));
+
+export default function TwoFingerPan() {
+  const r = useSharedValue(128);
+  const b = useSharedValue(128);
+
+  const pan = Gesture.Pan()
+    .onChange((event) => {
+      r.value = clampColor(r.value - event.changeY);
+      b.value = clampColor(b.value + event.changeX);
+    })
+    .runOnJS(true)
```

Comment:
Why does it need to run on the JS thread?

Reply:
Because this example is also available on iOS 😅

Reply:
What does it change?

Isn't making `clampColor` a worklet enough?

Reply:
You're right, it is. I don't think it needs a PR though 😅